woocommerce/woocommerce-ios
# Offline banners for views with cached data

Closes #4384
⚠️ This PR depends on #4986 so please make sure that that PR is review and merged first ⚠️ 

# Description
This PR configures offline banners for screens: My Store, Orders, Order Details, Review Order, Create Shipping Label, Products, Product Form, Reviews,  Review Detail.

For simplicity, offline banner is not added to every screen, but only to critical screens that display cached data.

# Changes
 - Updates UIImage+Woo extension with lightning icon for the offline banner.
 - Adds new extension for UIViewController to configure offline banner.
 - Configure offline banner for screens: My Store, Orders, Order Details, Review Order, Create Shipping Label, Products, Product Form, Reviews,  Review Detail.

# Demo

https://user-images.githubusercontent.com/5533851/133578461-c27944bf-3133-465a-b803-6e2db3008ebd.MP4



# Testing
In order to test connection properly, please build these changes to a real device. Network monitor doesn't work well on simulators. Follow these steps for testing:

1. On My Store tab, turn off both wifi and cellular data if it's available on your data. Notice that the offline banner shows up.
2. Navigate to Settings, notice that offline banner doesn't show here.
3. Back to My Store and turn wifi / cellular data on. Notice that the offline banner slides away.
4. Repeat the above steps for each of above screens and their subsequent unsupported screens in the same navigation stacks.

Update release notes:
